app/views/hotels/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1> <%= @hotel.name %>'s Page</h1>
+<br>
+<h4>All Guests who've stayed with us:</h4>
+<%# <ul>
+  <% @hotel.rooms.each do |room| %>
```

Comment:
This looks like a good start to the extension! Feel free to finish this out for good practice.